test/e2e-go/cli/goal/expect/goal_expect_test.go

```diff
@@ -20,10 +20,12 @@ import (
 	"testing"
 
 	"github.com/algorand/go-algorand/test/framework/fixtures"
+	"github.com/algorand/go-algorand/test/partitiontest"
 )
 
 // TestGoalWithExpect Process all expect script files with suffix Test.exp within the test/e2e-go/cli/goal/expect directory
 func TestGoalWithExpect(t *testing.T) {
+	partitiontest.PartitionTest(t)
```

Comment:
This should be be called here because it is already partitioned in `et.Run()` with better partitioning. 
Partitioning here will result in double partitioning, leaving many tests out.